keyhive_core/src/contact_card.rs

```diff
@@ -0,0 +1,32 @@
+use crate::{
+    principal::individual::{op::KeyOp, Individual},
+    util::hex,
+};
+
+#[derive(Debug)]
+pub struct ContactCard(KeyOp);
```

Comment:
We can ship this as-is for now, but I'm not sure that we need a keyop for a contact card. Prekeys exist purely as a broadcast mechanism, and ideally they get expired. We can instead do something like the following, as long as the user stores that X25519 key (which they do for prekeys)

```rust
struct ContactCard {
  id: IndividualId,
  prekey: Signed<ShareKey>
}
```